### ISSUE TITLE
JSON Upload

### ISSUE THREAD

**Issue by cristian-tamblay** [DashAISoftware/DashAI#97]
* Python version:3.8
* Operating System:Ubuntu 20.04

### Description

Daniel Roco tried to Upload a JSON Dataset on TabularClassificationTask. It failed and the endpoint responded with Unprocessable Entity.

**Comment by Oziel14** [comment]
From what I found out the error is caused by the lack of the "dataset_name" field inside the "dataloader_params". The problem comes from the fact that in the view (see picture) you can't provide a name for the dataset and you can't add one by default. This is the "dataloader_params" of the JsonDataloader that comes from the front end:

```
{"task_name":"TabularClassificationTask","dataloader":"JSONDataLoader","file":{},
"url":"","params":{},"splits":{"train_size":0.7,"test_size":0.2,"val_size":0.1,
"shuffle":true,"seed":0,"stratify":false}, "dataloader_params":{"data_key":"data","outputs_columns":[]},"splits_in_folders":false}
```
![Error1](https://github.com/DashAISoftware/DashAI/assets/70730520/904a8b16-6452-4784-ac96-fa55774a4656)

**Comment by maximi4321** [comment]
The problem was that `jsondataloader.json` was missing the `name` parameter.

It should be solved when #104  is merged.
